monitor/index.js

```diff
@@ -83,6 +83,16 @@ app.get('/alerts', async (req, res, next) => {
   }
 })
 
+app.get('/stuckTransfers', async (req, res, next) => {
```

Comment:
does it make sense to provide access to this endpoint only in case of the V1 bridge?

Reply:
Thanks for the suggestion! Done https://github.com/poanetwork/tokenbridge/pull/83/commits/369b90a6d5866408b3c679ca0f527166b999342c